addon/mixins/route.js

```diff
@@ -193,6 +199,7 @@ const RouteMixin = Ember.Mixin.create({
       currentPage: startingPage,
       _firstPageLoaded: false,
       _perPage: perPage,
+      _store: store,
```

Comment:
@redsandro So here is one example.  https://github.com/toranb/ember-cli-simple-store

I also created a custom functional store and I have find/findAll methods that wouldn't work with this.  Both of these just use Ember objects / JS objects.